main.py

```diff
@@ -2,6 +2,46 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres_to_create = ["Western", "Action", "Dramma"]
+    actors_to_create = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson")
+    ]
+
+    for genre in genres_to_create:
+        Genre.objects.create(name=genre)
+
+    for first_name, last_name in actors_to_create:
+        Actor.objects.create(first_name=first_name, last_name=last_name)
+
+    Genre.objects.filter(name="Dramma").update(name="Drama")
+
+    actors_to_update = {
```

Comment:
that is actually quiet complicated code just to update two records, but ok let it be :)

Reply:
I thought I needed to adapt to both options at the same time